versions.list

```diff
@@ -0,0 +1,5 @@
+created_at: 2015-08-23T17:22:53-07:00
+---
+GemA 1 e63fe62df0f1f459d2f70986f79745c6
+GemB 0.1.0,0.1.1,0.1.2,0.2.0,0.2.1,0.3.0,0.4.0,0.4.1,0.5.0,0.5.1,0.5.2,0.5.3 c54e4b7e14861a5d8c225283b75075f4
+
```

Comment:
It is for running the server on development. I'm not sure about a good place to put this file? If you point me the place, I can fix it!